ldk/csharp/OliveHelpsLDK/OliveHelpsLDK/LoopServiceFacade.cs

```diff
@@ -34,5 +75,55 @@ public IClipboardService Clipboard()
         {
             return _clipboard;
         }
+
+        public IFilesystemService Filesystem()
```

Comment:
To my eyes, it looks like these should be properties instead of methods. You could use a private setter too with auto-properties to get rid of the private fields above I think. Like, `public int MyProperty { get; private set; }`.

Reply:
Yes good point! I had forgotten that C# supported properties in their interfaces; so I'll switch that over.